### DISCUSSION TITLE
Getting Started macOS ARM

### DISCUSSION THREAD

**Discussion by Grayfox691** [jeffvli/feishin#129] [Q&A]
Howdy, I'm new to the community and new to the world of self-hosting. With that I'm beyond a novice working with apps like this. But, I'm a huge fan of the web application and want to support it if only by using it.

After mounting "0.1.1-mac-arm64.dmg and trying to run it I get an error in macOS stating

 ""fishing" is damaged and can’t be opened. You should move it to the Trash."

<img width="277" alt="Screenshot 2023-06-02 at 10 57 57 AM" src="https://github.com/jeffvli/feishin/assets/135363069/dc88bc3a-4465-4af8-800f-f9b253dd3e79">

I'm sure I've done something wrong and or stupid but could some one possibly point me in the right direction as how to fix this? I know applications like this often need other frameworks installed but I wasn't able to find any documentation as to if or what.

**Comment by jeffvli** [comment]
Check here for the fix https://github.com/jeffvli/feishin/issues/104

**Reply by Grayfox691** [reply]
Thank you kindly. Again, sorry for the silly question.
